### PR TITLE
Regression: WDIO test v2 and add more SDKs

### PR DESCRIPTION
**Pull request by samarsault**
Follow up from https://github.com/percy/cli/pull/1211
- Add testing for WDIO v2
- Add more SDKs - python, capybara and appium-js
- Fix PR branch name not getting updated on `issue_comment` event

Reference Run - https://github.com/percy/cli/actions/runs/4565663845

### CODE REVIEW

File: .github/workflows/sdk-regression.yml [hunk 1]
```diff
@@ -21,19 +21,30 @@ jobs:
...
           - percy-protractor
           - percy-selenium-js
           - percy-selenium-dotnet
           - percy-selenium-java
+          - percy-selenium-python
+          - percy-capybara
+          - percy-appium-js
           - gatsby-plugin-percy
     steps:
+      - uses: jungwinter/split@v2
```
Comment (itsjwala) [inline .github/workflows/sdk-regression.yml:34]: let's add a comment on L14, on how to use this, doesn't seem intuitive on first glance.

File: .github/workflows/sdk-regression.yml [hunk 2]
```diff
@@ -21,19 +21,30 @@ jobs:
...
         uses: convictional/trigger-workflow-and-wait@v1.6.5
         with:
           owner: percy
           repo: ${{ matrix.repo }}
           github_token: ${{ secrets.WORKFLOW_DISPATCH_ACTIONS_TOKEN }}
           workflow_file_name: test.yml
-          ref: master
-          client_payload: '{ "branch": "${{ github.event.pull_request.head.ref || github.ref_name }}"}'
+          ref: ${{ steps.split.outputs._1 || master }}
+          client_payload: '{ "branch": "${{ steps.comment-branch.outputs.head_ref || github.ref_name }}"}'
```
Comment (itsjwala) [inline .github/workflows/sdk-regression.yml:49]: I missed this before, why do we need `github.ref_name` this? we should fail the build if `comment-branch` fails, we don't want incorrect refs to trigger the build and have the checks green.

**Review by itsjwala — CHANGES_REQUESTED** [review]
@samarsault back to you, link previous PR in the description

File: .github/workflows/sdk-regression.yml (hunk 1 above)
Comment (itsjwala) [reply .github/workflows/sdk-regression.yml:34]: * documentation around not the actions itself, but how are we using it in ourcase, like what does stuff after `@ ` mean to us if someone is adding a new branch they'll read and straight away add that.
* currently reading `percy-webdriverio` and `percy-webdriverio@v2` doesn't sound very intuitive imo.

Comment (samarsault) [reply .github/workflows/sdk-regression.yml:35]: Added doc

File: .github/workflows/sdk-regression.yml [hunk 3]
```diff
@@ -21,19 +21,32 @@ jobs:
...
           owner: percy
           repo: ${{ matrix.repo }}
           github_token: ${{ secrets.WORKFLOW_DISPATCH_ACTIONS_TOKEN }}
           workflow_file_name: test.yml
-          ref: master
-          client_payload: '{ "branch": "${{ github.event.pull_request.head.ref || github.ref_name }}"}'
-          wait_interval: 15
+          ref: ${{ steps.split.outputs._1 || 'master' }}
+          client_payload: '{ "branch": "${{ steps.comment-branch.outputs.head_ref }}"}'
+          wait_interval: 15
```
Comment (itsjwala) [inline .github/workflows/sdk-regression.yml:52]: EOF

File: .github/workflows/sdk-regression.yml (hunk 1 above)
Comment (itsjwala) [reply .github/workflows/sdk-regression.yml:35]: can you move it to L14 maybe ( basically where we are using it) we're not concerned about the implementation details of the actions

**Review by itsjwala — APPROVED** [review]

**Review by itsjwala — APPROVED** [review]

**Pull request merged**